Float.FileDownloader.Tests/DownloadRequest.Tests.cs

```diff
@@ -72,5 +74,16 @@ public async Task TestServerNotFound()
             var request = new HttpRequestMessage(HttpMethod.Get, "http://www.floatfloatfloat.float/float");
             await Assert.ThrowsAsync<HttpRequestException>(async () => await DownloadRequest.Download(request, TempFilePath()));
         }
+
+        [Fact]
+        public async Task TestEmptyHandler()
+        {
+            var filePath = TempFilePath();
+            var request1 = new HttpRequestMessage(HttpMethod.Put, TestUriString());
+            await DownloadRequest.Download(request1, filePath);
+            var request2 = new HttpRequestMessage(HttpMethod.Get, TestUri());
```

Comment:
@EBusch I don't think you need to make two request here -- the purpose of this test is to verify that `DownloadRequest.Download` works as expected if the caller does _not_ specify a `HttpClientHandler`.

So the test only needs to do the following:
1. Create a test request (probably should be a GET request)
2. Invoke `DownloadRequest.Download` with the request and save the response to file
3. Assert that the file was created
4. Assert that the file contains the proper content